docs/src/developer/style.md

```diff
@@ -19,38 +19,68 @@ including its deviations from the Julia style guide.  In particular, note its po
   * [return statements](http://www.juliaopt.org/JuMP.jl/dev/style/#Return-statements-1)
   * [variable names](http://www.juliaopt.org/JuMP.jl/dev/style/#Use-of-underscores-within-names-1).
```

Comment:
We should remove this given the latest discussion on the subject.

Reply:
Resolved in https://github.com/NREL/PowerSystems.jl/pull/317/commits/e78c96eeb066bd5c291e2f455c079ebe66669d37.